tests: remove l7 stresstest

ianvernon opened the pull request:
Comment out the L7 stresstests. Since we don't test the output of this test, it just adds a lot of time onto the build. 

Signed-off by: Ian Vernon <ian@cilium.io>

joestringer reviewed (approved):
This seems reasonable to me, Issue #1851 will track the restoration of these tests at such a time that we could make better use of the output.

Commit:
tests: do not run L7 stresstests

Comment out L7 stresstests as their results currently are not being tested.
Add more logs with descriptive output for the test.

Signed-off by: Ian Vernon <ian@cilium.io>

aanm reviewed (dismissed):
Has it says in the header:
```
# This tests:
# - L7 stress test
# - Policies across k8s namespaces
```
It's better if you comment the L7 stress tests otherwise we won't test policies across k8s namespaces

tgraf commented on tests/k8s/tests/03-l7-stresstest.sh:
```diff
@@ -89,12 +89,14 @@ backend_svc_ip=$(kubectl get svc -n development | awk 'NR==2{print $2}')
 
-kubectl exec -n qa -i ${frontend_pod} -- wrk -t20 -c1000 -d60 "http://${backend_svc_ip}:80/"
-kubectl exec -n qa -i ${frontend_pod} -- ab -r -n 1000000 -c 200 -s 60 -v 1 "http://${backend_svc_ip}:80/"
+#log "running stress tests"
```
Please file a GitHub issue and add `GH-XXX` comment to make sure we don't forget to re-enable this.
